test/files/neg/patmat-sealed-reachable.scala

```diff
@@ -0,0 +1,18 @@
+// scalac: -Werror
```

Comment:
`//> using options -Werror`

You probably think we are so provincial and backwater here.